srml/staking/src/inflation.rs

```diff
@@ -147,19 +155,19 @@ mod test_inflation {
 	const x_ideal: f64 = 0.5;
 	const d: f64 = 0.05;
 
-	// Part left to 0.5
+	// Part left of `x_ideal`
```

Comment:
I know we don't fuss too much over regular `//` comments, but "part left" could be interpreted as "part remaining." Using "{Left, Right} part" would be better.